backend/core/connection_crud.py

```
@@ -0,0 +1,49 @@
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+from models.connection import Connection, ConnectionStatus
+from schemas.connection import ConnectionCreate
```

Ruff (0.8.2)

`schemas.connection.ConnectionCreate` imported but unused

Remove unused import: `schemas.connection.ConnectionCreate`

(F401)